tests/unit/test_analytics.py

```diff
@@ -1,6 +1,80 @@
 import pytest
+import mock
+import platform
+import json
 
-from dvc.analytics import Analytics
+from voluptuous import Schema, Any
+
+from dvc import analytics
+from dvc.cli import parse_args
+from dvc.utils.compat import str, builtin_str
+
+
+string = Any(str, builtin_str)
+
+
+@pytest.fixture
+def tmp_global_config(tmp_path):
+    """
+    Fixture to prevent modifying the actual global config
+    """
+    with mock.patch(
+        "dvc.config.Config.get_global_config_dir", return_value=str(tmp_path)
```

Comment:
Why not `fspath()`?

Reply:
removed the patch for `fspath` and now it doesn't work with pathlib.Path in 3.5: https://github.com/iterative/dvc/issues/2903

Reply:
It does for me, patch is only needed for Python 3.5 builtin `pathlib`, but pytests `tmp_path` uses `pathlib2`.

Reply:
@Suor we've been using `str` in all other places, so it is no biggie. We need to revisit that more carefully later and fix everywhere.

Reply:
@Suor , didn't new that pytest uses `pathlib2`!

Reply:
my bad, then, @Suor , I just assumed it was using builtin `pathlib`